.github/workflows/upload.yaml

```
@@ -0,0 +1,55 @@
+        uses: actions/checkout@main
+
+      - name: Setup AWS CLI
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Sync files to S3 bucket
+        run: |
+            aws s3 sync json s3://injective-lists/json --delete
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)